admin/data/sql/seeds/20190123182010.custom.familiarisation-check-form-dev-data.sql

```diff
@@ -0,0 +1,30 @@
+INSERT INTO mtc_admin.checkForm ([name], isDeleted, isLiveCheckForm, formData) VALUES ('MTC0103', 0, 0, '[
+        {
+            "f1" : 5,
+            "f2" : 6
+        },
+        {
+            "f1" : 10,
+            "f2" : 4
+        },
+        {
+            "f1" : 7,
```

Comment:
This is a minor non-blocking comment, but as the 8 and 9 times tables are generally considered the most difficult AIUI perhaps we should just use up something very easy for the developer version of the familiarisation check.  e.g. 2 times table or 5 or 10 times table?

Reply:
i assumed this familiarisation form had come from STA, or did you define it @GeorgeChatzigiannis ?

Reply:
@GuyHarwood missed that message for some reason. I defined it myself.

Reply:
I'll request one that is STA approved.

Reply:
@jon-shipley @GuyHarwood Apparently there is no way at this stage to acquire a familiarisation form from STA. I did update the one that is used with data from table 1 to 6.